dev-lisp/ecl/ecl-24.5.10.recipe

```diff
@@ -66,6 +77,13 @@ INSTALL()
 {
 	make install
 
+	prepareInstalledDevelLib \
+		libecl
+	fixPkgconfig
+
+	packageEntries devel \
+		$developDir
+
```

Comment:
You can enable 32bit (SECONDARY_ARCHITECTURES) for this, just change "$prefix/lib" into "$libDir" in the next part.

Reply:
To move the config and manpage (still had the changes on the 32bit install):

```
	packageEntries devel \
		$developDir \
		$binDir/ecl-config \
		$manDir/man1/ecl-config.1
```

Reply:
updated. thank you!